catalog/app/components/JsonEditor/State.js

```diff
@@ -0,0 +1,236 @@
+import * as R from 'ramda'
+import * as React from 'react'
+import Ajv from 'ajv'
+import isArray from 'lodash/isArray'
+import isUndefined from 'lodash/isUndefined'
+import toNumber from 'lodash/toNumber'
+
+export const ColumnIds = {
```

Comment:
I think it's a good practice to store such things in constants. Text editor's autocompletion helps to avoid mistypings.

I used CamelCase because this contstat meaning is enum, actually. And enums in typescript are CamelCased.

Reply:
ok, i see the point